index.js

```diff
@@ -10,10 +11,9 @@ function stringHelper () {
     const len = arguments.length
     for (let i = 0; i < len; i += 1) {
       let arg = arguments[i]
-      let type = classifier(arg)
-      if (type === 'function') this.cat(arg.apply(this))
-      else if (type === 'array') this.cat(...arg.flatten())
-      else if (type === 'undefined' || type === 'object') continue
+      if (isType('function', arg)) this.cat(arg.apply(this))
```

Comment:
Then, if Im almost always checking if something is a function, array or boolean, should I have 3 functions (isBool, isFunc, isArr)?